CONTRIBUTING.md

```diff
@@ -0,0 +1,102 @@
+# Contributing to OpenPubkey
+
+Welcome to OpenPubkey! We are so excited you are here. Thank you for your interest in contributing your time and expertise to the project. The following document details contribution guidelines.
+
+# Getting Started
+
+Whether you're addressing an open issue (or filing a new one), fixing a typo in our documentation, adding to core capabilities of the project, or introducing a new use case, anyone from the community is welcome here at OpenPubkey.
+
+## Include Licensing at the Top of Each File
+At the top of each file in your commit, please ensure the following is captured in a comment: 
+
+``` SPDX-License-Identifier: Apache-2.0 ```
```

Comment:
Are you trying to make a code block or use a fixed-width font here? If you are trying to make a code block, the triple backticks need to appear alone on a line. If you want fixed-width font, just use single ticks.

Reply:
Fixed! Thanks for catching that. : [07f484b](https://github.com/openpubkey/openpubkey/pull/55/commits/07f484bc3a428c4be89a5e2bbec3414703b183f5)